### PR TITLE
[Fix #1446] Move Channel targeting in the browser.

### LINKED ISSUE #1446: Move channel targeting to JEXL

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: snippets/base/migrations/0046_auto_20201027_1337.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.13 on 2020-10-27 13:37
```
Comment (pmac) [inline snippets/base/migrations/0046_auto_20201027_1337.py:1]: 1337 🤘

**Review by pmac — APPROVED** [review]
Looks good to me. Simplifying things feels so good 🌃

File: snippets/base/admin/fields.py [hunk 2]
```diff
@@ -31,6 +32,30 @@ def to_jexl(self, value):
...
+    def to_jexl(self, value):
+        if not value:
+            return None
+        values = value.split(';')
+
+        # When `release` is selected, also add `default`
+        if 'release' in values:
+            values.append('default')
+
+        return f'{self.attr_name} in {[x for x in values]}'
```
[the remark this answers is not in the document]
Comment (glogiotatidis) [reply snippets/base/admin/fields.py:56]: It produces a JEXL list and that's probably what makes it confusing. Example
`release in ['release', 'beta']`

File: snippets/base/migrations/0046_auto_20201027_1337.py (hunk 1 above)
Comment (glogiotatidis) [reply snippets/base/migrations/0046_auto_20201027_1337.py:1]: haha nice catch!

**Comment by glogiotatidis** [comment]
As always, thanks for you time!

File: snippets/base/admin/fields.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pmac) [reply snippets/base/admin/fields.py:56]: but isn't `values` already a list? wouldn't it be the same if you just used `{values}`?

**Pull request merged**